mmcv/device/npu/data_parallel.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Huawei Technologies Co., Ltd. All rights reserved.
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import sys
+
+import torch
+
+from mmcv.parallel import MMDataParallel
+from .scatter_gather import scatter_kwargs
+
+
+def _check_balance(*args, **kwargs):
+    return
+
+
+# Since we do not have a similar hardware unit multi_processor
+# on the NPU, the corresponding# devices_properties does not
+# have this property and cannot be checked. So we masked the
+# _check_balance function in DataParallel to make initialization pass.
+for m in sys.modules:
+    if m.startswith('torch') or 'mmcv' in m:
+        if getattr(sys.modules[m], '_check_balance', None) is not None:
```

Comment:
Can we use `hasattr` to check whether the modules have an attribute?

```python
if getattr(sys.modules[m], '_check_balance', None) is not None:
```
->
```python
if hasattr(sys.modules[m], '_check_balance'):
```

Reply:
fixed